File: 백준/Bronze/10870. 피보나치 수 5/피보나치 수 5.cc
#include <stdio.h>

int fibo(int num) {
	if (num == 0) return 0;
	if (num == 1) return 1;

	return fibo(num - 1) + fibo(num - 2);
}

int main(void) {
	int n = 0;
	if (scanf("%d", &n) != 1) return 0;
	if (0 > n || n > 20) return 0;

	int result = fibo(n);

	printf("%d\n", result);

	return 0;
}

File: 백준/Bronze/10870. 피보나치 수 5/피보나치 수 5.py
import sys

input = sys.stdin.readline
write = sys.stdout.write

def fibo(num: int):
    if num == 0: return 0
    if num == 1: return 1
    
    return fibo(num - 1) + fibo(num - 2)

def solve():
    n = int(input().rstrip())
    if 0 > n or n > 20: return
    
    result = fibo(n)
    
    write(str(result) + "\n")
    
if __name__ == "__main__":
    solve()